src/ontology/cl-edit.owl

```diff
@@ -29046,6 +29059,17 @@ SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023029 (indirect pathway medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Medium_spiny_neuron&oldid=1109726850"^^xsd:string) obo:IAO_0000115 obo:CL_4023029 "A medium spiny neuron that expresses dopamine type 2 receptors and projects to the external globus pallidus (GPe)."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023029 <https://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023029 "D1-MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4023029 "dopamine 1 medium spiny neuron")
```

Comment:
Don't see this synonym in provided dbxref and probably meant for direct pathway MSN.

Reply:
Same issue as noted [here](https://github.com/obophenotype/cell-ontology/pull/1752/files#r1048209727).